sql/src/main/java/io/crate/analyze/InsertFromValuesAnalyzedStatement.java

```diff
@@ -161,4 +163,12 @@ public void addGeneratedColumn(Reference reference) {
     public int numAddedGeneratedColumns() {
         return numAddedGeneratedColumns;
     }
+
+    public void bulkResponseIdxForId(String id, int bulkIdx) {
+        idToBulkResult.put(id, bulkIdx);
```

Comment:
I'd either have a method for both get and put or just expose the whole map, but not a mixture of both